backend/src/main/java/botobo/core/infrastructure/s3/S3Uploader.java

```diff
@@ -1,28 +1,28 @@
-package botobo.core.infrastructure;
+package botobo.core.infrastructure.s3;
 
-import botobo.core.exception.user.s3.FileConvertFailedException;
+import botobo.core.domain.user.User;
+import botobo.core.exception.user.s3.S3UploadFailedException;
 import com.amazonaws.services.s3.AmazonS3;
 import com.amazonaws.services.s3.model.DeleteObjectRequest;
 import com.amazonaws.services.s3.model.ObjectMetadata;
 import com.amazonaws.services.s3.model.PutObjectRequest;
+import com.google.common.net.MediaType;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Profile;
 import org.springframework.stereotype.Component;
 import org.springframework.web.multipart.MultipartFile;
 
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.FileOutputStream;
 import java.io.IOException;
+import java.io.InputStream;
 import java.util.Objects;
-import java.util.Optional;
 
+@Profile({"dev1", "dev2", "prod1", "prod2"})
```

Comment:
```suggestion
@Profile("!test & !local")
```
나중에 인스턴스가 추가돼도 수정하지 않아도 되게끔 바꿔주면 좋을 것 같아요.
아직 merge 되지 않은 제 PR에서 이렇게 적용했는데 dev나 prod에서 테스트는 못해봤지만, 아마 문제없을 거라 생각해요.

Reply:
suggested change안에 적힌게 안보여요.. ㅠㅠ

Reply:
피시에서는 보이는데.. 아까 슬랙에서 눌러서 살짝 봣을 때는 안 보이더라고요? 😓
```@Profile("!test & !local")```
이런 내용이었는데요
피케이가 dev1,2와 prod1,2를 하나도 통합한다고 하니 나중에 "dev", "prod" 로 바꾸면 돼서 그대로 두셔도 좋을 겉 같아요.

Reply:
네넹 !!!! 제가 요 피알에서 dev,prod로 수정할게요 !!!